metrics/web/test/auth/presentation/widgets/auth_form_test.dart

```diff
@@ -54,64 +51,71 @@ void main() {
     );
 
     testWidgets(
-        "email input shows an error message if a value is not a valid email",
-        (WidgetTester tester) async {
-      await tester.pumpWidget(_AuthFormTestbed(authNotifier: authNotifier));
-      await tester.enterText(emailInputFinder, 'notAnEmail');
+      "email input shows an error message if a value is not a valid email",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_AuthFormTestbed(authNotifier: authNotifier));
+        await tester.enterText(emailInputFinder, 'notAnEmail');
 
-      await tester.tap(submitButtonFinder);
-      await tester.pump();
+        await tester.tap(submitButtonFinder);
+        await tester.pump();
 
-      expect(find.text(AuthStrings.invalidEmailErrorMessage), findsOneWidget);
-    });
+        expect(find.text(AuthStrings.invalidEmailErrorMessage), findsOneWidget);
+      },
+    );
 
-    testWidgets("password input shows an error message if a value is empty",
-        (WidgetTester tester) async {
-      await tester.pumpWidget(_AuthFormTestbed(authNotifier: authNotifier));
+    testWidgets(
+      "password input shows an error message if a value is empty",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_AuthFormTestbed(authNotifier: authNotifier));
 
-      await tester.tap(submitButtonFinder);
-      await tester.pump();
+        await tester.tap(submitButtonFinder);
+        await tester.pump();
 
-      expect(
-        find.text(AuthStrings.passwordRequiredErrorMessage),
-        findsOneWidget,
-      );
-    });
+        expect(
+          find.text(AuthStrings.passwordRequiredErrorMessage),
+          findsOneWidget,
+        );
+      },
+    );
 
     testWidgets(
-        "signInWithEmailAndPassword method is called on tap on sign in button",
-        (WidgetTester tester) async {
-      final authNotifier = AuthNotifierMock();
+      "calls signInWithEmailAndPassword() method on tap on the sign in button",
```

Comment:
```suggestion
      "uses the AuthNotifier to sign in user",
```
or do we really need this test at all?